pkg/outputter.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 
 	"github.com/rs/zerolog"
```

Comment:
Is there a reason why we use different loggers in this repo? My little understanding tells me that we use Logrus. Why Zerolog again? I propose consistency.